src/main/kotlin/no/nav/klage/oppgave/domain/OppgaverQueryParams.kt

```diff
@@ -0,0 +1,15 @@
+package no.nav.klage.oppgave.domain
+
+data class OppgaverQueryParams(
```

Comment:
Jeg ser at denne brukes hele veien, fra Controlleren til OppgaveClient. Sånn rent bortsett fra kommentaren min i OppgaveController om at jeg ikke visste at man kunne bruke et objekt på denne måten, så er jeg kanskje litt skeptisk til å bruke samme klasse hele veien gjennom også. Jeg er vant med å definere rest-kontrakten med andre klasser enn det jeg bruker lenger inn i forretningslogikken og servicene og sånt, for å få et anti-corruption-layer og frikoble REST-apiet fra kjernefunksjonaliteten. Men jeg er åpen for diskusjon på det, synes du det er å overdrive?

Reply:
Jeg tenkte noe på den saken, men bare videreførte mønstret som var fra før (searchCriteria) inntil videre. Jeg prøver å dele opp så ser vi hvordan det blir :) Tenker også at det er en god idé.

Reply:
SearchCriteria ble opprettet i RestControlleren, men den var ikke med i selve rest-apiet/kontrakten. Så en endring i den klassen ville ikke ført til en endring i kontrakten.

Reply:
Stemmer. Har uansett laget en annen klasse som vi oversetter til og sender videre. Fikk lagt inn litt mere der også.